static/app/views/replays/details.tsx

```diff
@@ -41,6 +47,50 @@ export default function ReplayDetails() {
   });
   const {replay, replayRecord} = readerResult;
 
+  const {playlistStart, ...query} = useLocationQuery({
+    fields: {
+      cursor: decodeScalar,
+      environment: decodeList,
+      project: decodeList,
+      sort: decodeScalar,
+      query: decodeScalar,
+      playlistStart: decodeScalar,
+      statsPeriod: decodeScalar,
+      utc: decodeScalar,
+    },
+  });
+  const {getParamValue} = useUrlParams('sort');
+  const sortQuery = getParamValue();
+
+  const queryKey = useReplayListQueryKey({
+    options: {query: {...query, start: playlistStart, sort: sortQuery}},
+    organization,
+    queryReferrer: 'replayList',
+  });
+  const {data} = useApiQuery<{
+    data: ReplayListRecord[];
+    enabled: true;
+  }>(queryKey, {staleTime: 0});
```

Comment:
What happens when we visit replay details from a non-replay index source? (e.g. from issue details)